src/main/java/core/basesyntax/converter/StringToFruitTransactionConverter.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+public class StringToFruitTransactionConverter implements Function<String, FruitTransaction> {
+
+    @Override
+    public FruitTransaction apply(String s) {
+        String[] arr = s.split(",");
+        if (arr.length != 3) {
+            throw new RuntimeException("Cannot convert string [" + s + "] to FruitTransaction");
+        }
+        return new FruitTransaction(
```

Comment:
Ensure that the `FruitTransaction` constructor is correctly handling the input parameters. Since `FruitTransaction` is expected to represent a fruit transaction with an operation, fruit name, and quantity, verify that the input data is valid and meaningful.